RemovedInPytest4Warning: MarkInfo objects are deprecated

This warning is currently ignored as per astropy/astropy#7490 but still should be addressed at some point?

```
    def pytest_runtest_setup(item):
    
        remote_data = item.get_marker('remote_data')
        internet_off = item.get_marker('internet_off')
    
        remote_data_config = item.config.getvalue("remote_data")
    
        if remote_data is not None and internet_off is not None:
            raise ValueError("remote_data and internet_off are not compatible")
    
        if remote_data is not None:
>           source = remote_data.kwargs.get('source', 'any')
E           _pytest.deprecated.RemovedInPytest4Warning: MarkInfo objects are deprecated as they contain merged marks which are hard to deal with correctly.
E           Please use node.get_closest_marker(name) or node.iter_markers(name).
E           Docs: https://docs.pytest.org/en/latest/mark.html#updating-code
/home/travis/miniconda/envs/test/lib/python3.6/site-packages/pytest_remotedata/plugin.py:68: RemovedInPytest4Warning
```

https://github.com/astropy/pytest-remotedata/blob/v0.3.0/pytest_remotedata/plugin.py#L76

--- a/pytest_remotedata/plugin.py
+++ b/pytest_remotedata/plugin.py
@@ -5,6 +5,7 @@ make use of online data.
 """
 import pytest
 from .disable_internet import turn_off_internet, turn_on_internet
+from distutils.version import StrictVersion
 
 
 def pytest_addoption(parser):
@@ -61,8 +62,12 @@ def pytest_unconfigure():
 
 def pytest_runtest_setup(item):
 
-    remote_data = item.get_closest_marker('remote_data')
-    internet_off = item.get_closest_marker('internet_off')
+    if StrictVersion(pytest.__version__) < StrictVersion("3.6"):
+        remote_data = item.get_marker('remote_data')
+        internet_off = item.get_marker('internet_off')
+    else:
+        remote_data = item.get_closest_marker('remote_data')
+        internet_off = item.get_closest_marker('internet_off')
 
     remote_data_config = item.config.getvalue("remote_data")
 

--- a/pytest_remotedata/plugin.py
+++ b/pytest_remotedata/plugin.py
@@ -61,8 +61,8 @@ def pytest_unconfigure():
 
 def pytest_runtest_setup(item):
 
-    remote_data = item.get_marker('remote_data')
-    internet_off = item.get_marker('internet_off')
+    remote_data = item.get_closest_marker('remote_data')
+    internet_off = item.get_closest_marker('internet_off')
 
     remote_data_config = item.config.getvalue("remote_data")
 

--- a/setup.cfg
+++ b/setup.cfg
@@ -1,4 +1,4 @@
-[pytest]
+[tool:pytest]
 minversion = 3.1
 testpaths = tests
 remote_data_strict = true

